strato_query/base_API_query.py

```diff
@@ -689,7 +689,7 @@ def submit_query(cls,
         The result from the request, either a pandas dataframe or a dict with dataframes as values
         """
         assert query_params is None or isinstance(query_params, APIQueryParams)
-        assert queries_params is None or isinstance(queries_params, APIQueryParams)
+        assert queries_params is None or isinstance(queries_params, dict)
```

Comment:
Do these entries in the `dict` need to be `APIQueryParams`? If so, can you check `queries_params.values()`?

Reply:
They do, and yes.